### PR TITLE
KHP3-6174 - (feat) added a UI for the showing prices

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

-  added  UI to support procedure, lab and radiology prices on the order basket
https://thepalladiumgroup.atlassian.net/browse/KHP3-6174

## Screenshots
example of procedure
basket

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-billing-app/src/billable-services/billiable-item/test-order/lab-oder.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@use '@carbon/styles/scss/type';
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/layout';
+@use '@carbon/colors';
+@import '@openmrs/esm-styleguide/src/vars';
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/billable-services/billiable-item/test-order/lab-oder.scss:5]:
```suggestion

```

File: packages/esm-billing-app/src/billable-services/billiable-item/test-order/lab-order.component.tsx [hunk 2]
```diff
@@ -12,23 +23,58 @@ type LabOrderProps = {
...
 
   if (isLoading) {
-    return null;
+    return (
+      <InlineNotification
+        kind="info"
+        title={t('loadingPrices', 'Loading prices')}
+        subtitle={t('pleaseWait', 'Please wait...')}
+        lowContrast
+      />
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/billable-services/billiable-item/test-order/lab-order.component.tsx:36]:
```suggestion
      <InlineLoading status="active" iconDescription="Loading" description="Loading data..." />
```

File: packages/esm-billing-app/src/billable-services/billiable-item/test-order/lab-order.component.tsx [hunk 3]
```diff
@@ -12,23 +23,58 @@ type LabOrderProps = {
...
+    );
   }
 
-  return <p>{billItems}</p>;
+  return (
+    <Tile id="" className={styles.prices}>
+      <div className={styles.listContainer}>
+        <StructuredListWrapper isCondensed>
+          A
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/billable-services/billiable-item/test-order/lab-order.component.tsx:55]:
```suggestion
          
```

**Review by donaldkibet — CHANGES_REQUESTED** [review]

**Comment by ojwanganto** [comment]
@its-kios09 - please address the comments

**Comment by its-kios09** [comment]
> @its-kios09 - please address the comments

I have already address them

**Comment by makombe** [comment]
@its-kios09 Address the failing lint issues. Thanks

**Comment by its-kios09** [comment]
> @its-kios09 Address the failing lint issues. Also, can you update the video to reflect the current status. Thanks



> @its-kios09 Address the failing lint issues. Also, can you update the video to reflect the current status. Thanks



> @its-kios09 Address the failing lint issues. Also, can you update the video to reflect the current status. Thanks

I have clean it

**Review by makombe — APPROVED** [review]
LTGM. Thanks @its-kios09

**Pull request merged**